iib/workers/tasks/build.py

```diff
@@ -952,6 +952,20 @@ def get_image_label(pull_spec, label):
     return get_image_labels(pull_spec).get(label)
 
 
+def create_dockerconfig(registry_auths):
+    home = os.path.expanduser('~')
+    dockerconfig_path = os.path.join(home, '.docker/config.json')
```

Comment:
@lcarva do we want to replace it for https://github.com/release-engineering/iib/blob/master/iib/workers/config.py#L15 ??

Reply:
Let's use the existing `set_registry_token` context manager: https://github.com/release-engineering/iib/blob/c8b668b95d41b9d4807a279a1b1154f2296930fe/iib/workers/tasks/utils.py

Reply:
I added adjustment to `set_registry_token` then, hope this is close to what you had in mind.

Reply:
If it looks usable it's probably worth to rename that method.